src/software/pipeline/main_panoramaWarping.cpp

```diff
@@ -124,6 +125,8 @@ int aliceVision_main(int argc, char** argv)
         "Max Panorama Width in pixels.")("percentUpscale",
                                          po::value<int>(&percentUpscale)->default_value(percentUpscale),
                                          "Percentage of upscaled pixels.")(
+        "outputColorSpace", po::value<image::EImageColorSpace>(&outputColorSpace)->default_value(outputColorSpace),
```

Comment:
I would call it workingColorspace to indicate that the process will be done in that colorspace, then it outputs its result without convertion.

Reply:
Done